### PR TITLE
Refactor Board (Removing PlayerPositions Map)

### PR DESCRIPTION
**Pull request by derrcodes**
Changes:

- removal of PlayerPosition instance attribute (includes the removal of getters and setters)
- removal of the second Board constructor with the parameter used to set the value of the PlayerPositions map during initialization.

### CODE REVIEW

**Comment by benxu3** [comment]
I will make these changes once the load-tests-docs PR gets merged in. I made some changes there assuming that Board no longer keeps track of player position.

File: src/main/java/Entities/Board.java [hunk 1]
```diff
@@ -8,7 +8,6 @@
 
     List<Player> players;
     List<Cell> cells;
-    Map <Player, Integer> playerPositions = new HashMap<Player, Integer>();
```
Comment (harshkpatel) [inline src/main/java/Entities/Board.java:11]: good choice, let players handle their own positions

**Review by harshkpatel — APPROVED** [review]
Good refactor removing this since players were already keeping track.

**Review by giaphuongd — APPROVED** [review]
Good job removing the HashMap storing the player positions. Player class stores this information so it was redundant. Nice work cleaning up the code to match this removal.

File: src/main/java/Interactors/PerformActionSpaceCardInteractor.java [hunk 2]
```diff
@@ -114,7 +110,6 @@ private String getOutOfJailAction(Player player, Card card) {
      * Aggregates all the actions of the card and the main perform action method
      * @param player the player
      * @param board the board
-     * @param card the card
```
Comment (susannah-c) [inline src/main/java/Interactors/PerformActionSpaceCardInteractor.java:117]: I think you're missing the parameter for actionSpace2.

**Review by susannah-c — COMMENTED** [review]
The decision to let Player keeps track of their position is great, otherwise, it would have been redundant. Good job on following up on refactoring the methods and tests.

**Pull request merged**